COMMITTING.md

```diff
@@ -0,0 +1,55 @@
+# WHATWG committer guidelines
+
+## Branches
```

Comment:
This seems like reasonable advice, but I'd put it at the end, as the problems with PRs are almost exclusively with commit messages. We don't care too much about branches since they disappear quickly (if all goes well).